images/pulsar-functions-java-runner/pulsarctl.Dockerfile

```diff
@@ -53,4 +56,26 @@ ENV java.io.tmpdir=/pulsar/tmp/
 
 WORKDIR /pulsar
 
+# Copy the kinesis_producer native executable compiled for Alpine musl to the pulsar-all image
+# This is required to support the Pulsar IO Kinesis sink connector
+COPY --from=pulsar --chown=$UID:$GID /opt/amazon-kinesis-produce* /opt/amazon-kinesis-producer
+# Set the environment variable to point to the kinesis_producer native executable
+ENV PULSAR_IO_KINESIS_KPL_PATH=/opt/amazon-kinesis-producer/bin/kinesis_producer
+# Install the required dependencies for the kinesis_producer native executable
+USER 0
+RUN apk update && apk add --no-cache \
+    brotli-libs \
+    c-ares \
+    libcrypto3 \
+    libcurl \
+    libgcc \
+    libidn2 \
+    libpsl \
+    libssl3 \
+    libunistring \
+    nghttp2-libs \
+    zlib \
+    zstd-libs \
+    libuuid
+
```

Comment:
why is this needed if `pulsar-functions-pulsarctl-runner-base:latest` is the base image with similar changes?